webapp/app/js/directives/clonejobs.js

```diff
@@ -1069,6 +1175,84 @@ treeherder.directive('thCloneJobs', [
             tableEl.append(row);
         }
     };
+    
+    var traceParent = function(el, level){
```

Comment:
We could probably do without this function.  It looks like you always just trace a single parent higher.  So couldn't you just call ``el.parent()`` instead?